src/axiosCromwell.ts

```diff
@@ -1,10 +1,51 @@
 import axios from 'axios';
+import { useDispatch } from "react-redux";
+import { sessionExpired } from "./store/auth/actions";
 
 const axiosCromwell = axios.create({
-  baseURL: 'http://localhost:3001/',
+  baseURL: 'http://localhost:8080/',
   headers: {
     'Content-Type': 'application/json'
   }
 });
 
+axiosCromwell.interceptors.request.use(
+  config => {
+    const accessToken = localStorage.getItem('accessToken');
+    if (accessToken) {
+      config.headers["Authorization"] = accessToken;
+    }
+    return config;
+  },
+  error => {
+    Promise.reject(error);
+  }
+);
+
+axiosCromwell.interceptors.response.use(
+  response => {
+    return response;
```

Comment:
I'd added an identity function somewhere in the utils module ;)

Reply:
Ahah, I also thought about it and about adding fp-ts for this purpose :smile: 
But I guess this case is not a reason for adding fp-ts (I'm going to make a separate request in the future where we will be using more functional constructs).

So yes, let's added our own identity function.